modules/services/emacs.nix

```diff
@@ -24,18 +24,23 @@ in
         description = "This option specifies the emacs package to use.";
       };
 
+      exec = mkOption {
```

Comment:
n/m I think just the binary name makes more sense then.

The idea was that you'd only have to set one option. A better approach would be to add the command name to the package and do something like this.

```nix
  "${cfg.package}/${cfg.package.execName or cfg.exec}"
```